app/assets/v2/css/grants/fund.css

```diff
@@ -3,6 +3,16 @@
   color: #0D0764;
 }
 
+.banner-img {
+  max-height: 22rem;
+  overflow-y: hidden;
+}
+
+.banner-img img {
+  height: auto;
+  width: 100%;
+}
```

Comment:
you can use `object-fit: cover;` to be sure is always is going to cover the space.